pkg/operator/baremetal_pod.go

```diff
@@ -528,12 +554,14 @@ func createContainerMetal3IronicConductor(config *OperatorConfig, baremetalProvi
 		VolumeMounts: []corev1.VolumeMount{
 			sharedVolumeMount,
 			inspectorCredentialsMount,
+			rpcCredentialsMount,
```

Comment:
Conductor is only the server side, and should not have client credentials.

Reply:
Conductor appears to need the credentials to talk to itself, I believe @dtantsur is currently working on removing this requirement.

Reply:
TIL, thanks.
Looks like this might be the culprit, it's used by the drivers: https://opendev.org/openstack/ironic/src/branch/master/ironic/conductor/utils.py#L833-L848

Reply:
I am indeed working on removing this requirement, but it's going to take substantial time.